CONTRIBUTING.md

```diff
@@ -1,64 +1,90 @@
-# Contributing Guide
+# Contributing guide
 
-## 🚀 Get Started
+Thank you for your interest in contributing to [nodejs.dev](https://nodejs.dev). Before you proceed, briefly go through the following:
 
-* Fork the [nodejs.dev repo][] using the Fork button on the top right
+  [Code of Conduct](https://github.com/nodejs/node/blob/HEAD/CODE_OF_CONDUCT.md)
 
-* Clone your fork using SSH, GitHub CLI, or HTTPS
+* [🚀 Getting started](#-getting-started)
+  * [Vocabulary](#vocabulary)
+  * [📝 Commit message guidelines](#-commit-message-guidelines)
+  * [📜 Pull Request Policy](#-pull-request-policy)
+    * [Before merging](#before-merging)
+    * [When merging](#when-merging)
+  * [Becoming a collaborator](#becoming-a-collaborator)
+  * [🐛 Debugging failing checks](#-debugging-failing-checks)
+  * [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+  * [Remarks](#remarks)
+
+## 🚀 Getting started
+
+1. Click the fork button in the top right to clone the [nodejs.dev repo][]
+
+2. Clone your fork using SSH, GitHub CLI, or HTTPS.
 
   ```bash
   git clone git@github.com:<GITHUB_ID>/nodejs.dev.git # SSH
   gh repo clone <GITHUB_ID>/nodejs.dev # GitHub CLI
   git clone https://github.com/<GITHUB_ID>/nodejs.dev.git # HTTPS
   ```
 
-* Change into the nodejs.dev project directory
+3. Change into the nodejs.dev directory.
 
   ```bash
   cd nodejs.dev
   ```
 
-* Add nodejs/nodejs.dev as your upstream remote branch
+4. Create a remote for keeping your fork as well as your local clone up-to-date.
 
   ```bash
   git remote add upstream git@github.com:nodejs/nodejs.dev.git
   ```
 
-* Create a new branch for your awesome work
+5. Create a new branch for your work.
 
   ```bash
-  git checkout -b <BRANCH_NAME>
+  git checkout -b name-of-your-branch
   ```
 
-* Confirm tests, linting, and formatting are passing. See [here](#-debugging-failing-checks) to fix failures
+6. Run the following to install the dependencies and start a local preview of your work.
 
   ```bash
-  npm test # Runs formatter and linter also
+  npm install # installs this project's dependencies
+  npm start # starts a preview of your local changes
   ```
 
-* Commit your work. See [Commit Guidelines](#-commit-message-guidelines)
+7. Perform a merge to sync your current branch with the upstream branch.
 
-* Push to your branch
+ ```bash
+git fetch upstream
+git merge upstream/main
+```
 
-  ```bash
-  git push -u origin <YOUR_BRANCH>
-  ```
+8. Run `npm test` to confirm that tests, linting, and formatting are passing. See [here](#-debugging-failing-checks) to fix failures. Ensure that your tests are passing before making a PR.
 
-* Open a pull request. See [PR Policy](#-pull-request-policy)
+```bash
+npm test
+```
 
-## Vocabulary
+9. Once you're happy with your changes, add and commit them to your branch,
+then push the branch to your fork.
+
+    ```sh
+    cd ~/nodejs.dev
+    git add .
+    git commit
```

Comment:
probably want to do `git commit -m "some message"` so newer devs don't get stuck in vim